packages/components/src/Actions/Action/Action.component.js

```diff
@@ -24,12 +41,12 @@ import getPropsFrom from '../../utils/getPropsFrom';
 function Action(props) {
 	const {
 		bsStyle,
-		icon,
 		inProgress,
 		hideLabel,
 		label,
 		link,
 		model,
+		reverseDisplay,
```

Comment:
I would prefer iconPosition with a default to left or iconAfterText with default to false.